### PR TITLE
Learning about the CSS Sass extension!

### PR DESCRIPTION
**Pull request by ygordeoliveira**
Sass is a CSS extension that has many special features making the development process simpler and more efficient.

Sass aims to facilitate the creation of code in CSS, it brings features that CSS does not have such as: variables, mixin functions, loops, nesting element style rules and much more.

### CODE REVIEW

File: sass/apply-a-style-until-a-condition-is-met-with-@while/apply-a-style-until-a-condition-is-met-with-while.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+    <html lang="pt-br">
```
Comment (rodrigobmedeiros) [inline sass/apply-a-style-until-a-condition-is-met-with-@while/apply-a-style-until-a-condition-is-met-with-while.html]: the full HTML appears to be indented one level inwards

File: sass/create-reusable-css-with-mixins/create-reusable-css-with-mixins.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+    <html lang="pt-br">
```
Comment (rodrigobmedeiros) [inline sass/create-reusable-css-with-mixins/create-reusable-css-with-mixins.html]: the full HTML appears to be indented one level inwards

File: sass/extend-one-set-of-css-styles-to-another-element/extend-one-set-of-css-styles-to-another-element.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+    <html lang="pt-br">
```
Comment (rodrigobmedeiros) [inline sass/extend-one-set-of-css-styles-to-another-element/extend-one-set-of-css-styles-to-another-element.html]: the full HTML appears to be indented one level inwards

File: sass/nest-css-with-sass/nest-css-with-sass.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+    <html lang="pt-br">
```
Comment (rodrigobmedeiros) [inline sass/nest-css-with-sass/nest-css-with-sass.html]: the full HTML appears to be indented one level inwards

File: sass/store-data-with-sass-variables/store-data-with-sass-variables.html [hunk 5]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+    <html lang="pt-br">
```
Comment (rodrigobmedeiros) [inline sass/store-data-with-sass-variables/store-data-with-sass-variables.html]: the full HTML appears to be indented one level inwards

File: sass/use-@each-to-map-over-items-in-a-list/use-each-to-map-over-items-in-a-list.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+    <html lang="pt-br">
```
Comment (rodrigobmedeiros) [inline sass/use-@each-to-map-over-items-in-a-list/use-each-to-map-over-items-in-a-list.html]: the full HTML appears to be indented one level inwards

File: sass/use-@for-to-create-a-sass-loop/use-for-to-create-a-sass-loop.html [hunk 7]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+   <html lang="pt-br">
```
Comment (rodrigobmedeiros) [inline sass/use-@for-to-create-a-sass-loop/use-for-to-create-a-sass-loop.html]: the full HTML appears to be indented one level inwards

File: sass/use-@if-and-@else-to-add-logic-to-your-styles/use-if-and-else-to-add-logic-to-your-styles.html [hunk 8]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+    <html lang="pt-br">
```
Comment (rodrigobmedeiros) [inline sass/use-@if-and-@else-to-add-logic-to-your-styles/use-if-and-else-to-add-logic-to-your-styles.html]: the full HTML appears to be indented one level inwards

**Review by rodrigobmedeiros — CHANGES_REQUESTED** [review]

File: sass/apply-a-style-until-a-condition-is-met-with-@while/apply-a-style-until-a-condition-is-met-with-while.html (hunk 1 above)
Comment (ygordeoliveira) [reply sass/apply-a-style-until-a-condition-is-met-with-@while/apply-a-style-until-a-condition-is-met-with-while.html]: The complete HTML is four levels inward.

With the child elements of the html element which are the head and body with four levels of indentation.

And the child elements of head and body also have four levels of indentation.

File: sass/create-reusable-css-with-mixins/create-reusable-css-with-mixins.html (hunk 2 above)
Comment (ygordeoliveira) [reply sass/create-reusable-css-with-mixins/create-reusable-css-with-mixins.html]: The complete HTML is four levels inward.

With the child elements of the html element which are the head and body with four levels of indentation.

And the child elements of head and body also have four levels of indentation.

File: sass/extend-one-set-of-css-styles-to-another-element/extend-one-set-of-css-styles-to-another-element.html (hunk 3 above)
Comment (ygordeoliveira) [reply sass/extend-one-set-of-css-styles-to-another-element/extend-one-set-of-css-styles-to-another-element.html]: The complete HTML is four levels inward.

With the child elements of the html element which are the head and body with four levels of indentation.

And the child elements of head and body also have four levels of indentation.

File: sass/nest-css-with-sass/nest-css-with-sass.html (hunk 4 above)
Comment (ygordeoliveira) [reply sass/nest-css-with-sass/nest-css-with-sass.html]: The complete HTML is four levels inward.

With the child elements of the html element which are the head and body with four levels of indentation.

And the child elements of head and body also have four levels of indentation.

File: sass/store-data-with-sass-variables/store-data-with-sass-variables.html (hunk 5 above)
Comment (ygordeoliveira) [reply sass/store-data-with-sass-variables/store-data-with-sass-variables.html]: The complete HTML is four levels inward.

With the child elements of the html element which are the head and body with four levels of indentation.

And the child elements of head and body also have four levels of indentation.

File: sass/use-@each-to-map-over-items-in-a-list/use-each-to-map-over-items-in-a-list.html (hunk 6 above)
Comment (ygordeoliveira) [reply sass/use-@each-to-map-over-items-in-a-list/use-each-to-map-over-items-in-a-list.html]: The complete HTML is four levels inward.

With the child elements of the html element which are the head and body with four levels of indentation.

And the child elements of head and body also have four levels of indentation.

File: sass/use-@for-to-create-a-sass-loop/use-for-to-create-a-sass-loop.html (hunk 7 above)
Comment (ygordeoliveira) [reply sass/use-@for-to-create-a-sass-loop/use-for-to-create-a-sass-loop.html]: The complete HTML is four levels inward.

With the child elements of the html element which are the head and body with four levels of indentation.

And the child elements of head and body also have four levels of indentation.

File: sass/use-@if-and-@else-to-add-logic-to-your-styles/use-if-and-else-to-add-logic-to-your-styles.html (hunk 8 above)
Comment (ygordeoliveira) [reply sass/use-@if-and-@else-to-add-logic-to-your-styles/use-if-and-else-to-add-logic-to-your-styles.html]: The complete HTML is four levels inward.

With the child elements of the html element which are the head and body with four levels of indentation.

And the child elements of head and body also have four levels of indentation.

**Review by rodrigobmedeiros — APPROVED** [review]

**Pull request merged**